app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,5 @@
  *
  *= require_self
  *= require select2
- *= require jquery.ui.datepicker
```

Comment:
Yep!
> On 3 Jun 2016, at 17:16, Bandad <notifications@github.com> wrote:
> 
> In app/assets/stylesheets/application.css <https://github.com/bannio/mars/pull/4#discussion_r65734652>:
> 
> > @@ -10,6 +10,5 @@
> >   *
> >   *= require_self
> >   *= require select2
> > - *= require jquery.ui.datepicker
> don’t know! Is that a change I made? Bob Bannister +44 7977 500360
> … <x-msg://99/#>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
>  <https://github.com/bannio/mars/pull/4#discussion_r65733634> <https://github.com/bannio/mars/pull/4/files/b6897336332661089e43cb007e6147a71de7b7f6#r65733634> <https://github.com/notifications/unsubscribe/AE50A6dC0Jx-eKMxG9bcV6CbIegSlMjRks5qIFG9gaJpZM4IlQlN>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/bannio/mars/pull/4/files/b6897336332661089e43cb007e6147a71de7b7f6#r65734652>, or mute the thread <https://github.com/notifications/unsubscribe/ABwOHWbfkSxhSvhWUYA2D8nO3fq_aD97ks5qIFNQgaJpZM4IlQlN>.
>